packages/account/src/Components/currency-selector/currency-selector.tsx

```diff
@@ -224,13 +227,13 @@ const CurrencySelector = observer(
                                     is_disabled={is_desktop}
                                 >
                                     <ThemedScrollbars height={height}>
-                                        {!!reorderCurrencies(fiat)?.length && (
```

Comment:
Why are you removing the order? is this part of the scope of your card?

Reply:
In this line the component is to be re-rendered only when the there are list of fiat-currencies. So we need not get the reordered list. I removed that as it was an un-necessary function call